### PR TITLE
[5G: MVC][component: agw][type: enhancement] AMF-Core

### PR DESCRIPTION
**Pull request by chandra-77**
## Summary

This is backporting of PR #4595 

## Test Plan

5G integration are passed. 
s1ap integration tests are passed.
## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/c/oai/lib/3gpp/3gpp_38.413.h [hunk 1]
```diff
@@ -191,7 +191,7 @@ typedef struct PDU_Session_resource_setup_request_s {
 // Direction: NG-RAN node → AMF
 typedef struct PDU_Session_resource_setup_response_s {
   Ngap_Message_Type_t Ngap_Message_Type;
-  amf_ue_ngap_id_t
+  amf_ue_ngap_id_ty
```
Comment (ulaskozat) [inline lte/gateway/c/oai/lib/3gpp/3gpp_38.413.h]: why is this changed? how does this compile?

Comment (chandra-77) [reply lte/gateway/c/oai/lib/3gpp/3gpp_38.413.h]: Fixed
